.github/workflows/cypress.yml

```diff
@@ -1,5 +1,11 @@
 name: Cypress Tests
-on: push
+on:
+  workflow_call:
+    secrets:
+      CYPRESS_TEST_USER_EMAIL:
+        required: true
+      CYPRESS_TEST_USER_PASS:
+        required: true
```

Comment:
`workflow_call` means it's called my other workflows (instead of on code push).
As a result, it doesn't have its own env or secrets so those have to be passed explicitly.